### PR TITLE
New Script: Glance

### PR DESCRIPTION
**Pull request by quantumryuu**
> [!NOTE]
> We are meticulous when it comes to merging code into the main branch, so please understand that we may reject pull requests that do not meet the project's standards. It's never personal. Also, game-related scripts have a lower chance of being merged.

## Description
Created install scripts for [Glance](https://github.com/glanceapp/glance/)
Provide a summary of the changes made and/or reference the issue being addressed.

Fixes # (issue)
#256 
[Project Board](https://github.com/orgs/community-scripts/projects/9/views/1?filterQuery=glance&pane=issue&itemId=87952082)
## Type of change
Please check the relevant option(s):

- [ ] Bug fix (non-breaking change that resolves an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (a fix or feature that would cause existing functionality to change unexpectedly)
- [x] New script (a fully functional and thoroughly tested script or set of scripts.)

## Prerequisites
The following efforts must be made for the PR to be considered. Please check when completed:
- [x] Self-review performed (I have reviewed my code, ensuring it follows established patterns and conventions)
- [x] Testing performed (I have tested my changes, ensuring everything works as expected)
- [x] Documentation updated (I have updated any relevant documentation)

## Additional Information (optional)
Provide any additional context or screenshots about the feature or fix here.
YML example taken from official repo.

## Related Pull Requests / Discussions

If there are other pull requests or discussions related to this change, please link them here:
- Related PR #

### CODE REVIEW

**Review by MickLesk — COMMENTED** [review]
some little comments

File: ct/glance.sh [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    RELEASE=$(curl -s https://api.github.com/repos/glanceapp/glance/releases/latest | grep "tag_name" | awk '{print substr($2, 3, length($2)-4) }')
+  if [[ ! -f /opt/${APP}_version.txt ]] || [[ "${RELEASE}" != "$(cat /opt/${APP}_version.txt)" ]]; then
+    msg_info "Stopping Service"
+    systemctl stop glance
+    msg_ok "Stopped Service"
+
+    msg_info "Updating ${APP} to v${RELEASE}"
+    cd /opt
+    wget -q https://github.com/glanceapp/glance/releases/download/v${RELEASE}/glance-linux-amd64.tar.gz 
+    rm -rf /opt/glance/glance
```
[the remark this answers is not in the document]
Comment (kristocopani) [reply ct/glance.sh:74]: Nope, /opt/glance/glance is a binary.
User data is stored in the /opt/glance/glance.yml file 😃

File: install/glance-install.sh [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+verb_ip6
+catch_errors
+setting_up_container
+network_check
+update_os
+
+msg_info "Installing Dependencies"
+$STD apt-get install -y \
+    curl \
+    wget \
```
[the remark this answers is not in the document]
Comment (kristocopani) [reply install/glance-install.sh:19]: Yup, copy paste of another script. Will fix it in a bit!

File: install/glance-install.sh [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+    mc 
+msg_ok "Installed Dependencies"
+
+
+msg_info "Installing Glance"
+RELEASE=$(curl -s https://api.github.com/repos/glanceapp/glance/releases/latest | grep "tag_name" | awk '{print substr($2, 3, length($2)-4) }')
+cd /opt
+wget -q https://github.com/glanceapp/glance/releases/download/v${RELEASE}/glance-linux-amd64.tar.gz
+mkdir /opt/glance
```
[the remark this answers is not in the document]
Comment (kristocopani) [reply install/glance-install.sh:28]: Is -p really needed? I mean /opt exists and I just create the folder called "glance" to add the config yaml file and the executable.
If needed I will add it of course

File: install/glance-install.sh [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+tar -xzf glance-linux-amd64.tar.gz -C /opt/glance
+cat <<EOF >/opt/glance/glance.yml
+pages:
```
[the remark this answers is not in the document]
Comment (kristocopani) [reply install/glance-install.sh:31]: Will test and commit when ready

File: install/glance-install.sh [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+WorkingDirectory=/opt/glance
+ExecStart=/opt/glance/glance --config /opt/glance/glance.yml
+TimeoutStopSec=20
+KillMode=process
+Restart=on-failure
+
+[Install]
+WantedBy=multi-user.target" >$service_path
+
+systemctl enable --now -q glance.service
```
[the remark this answers is not in the document]
Comment (kristocopani) [reply install/glance-install.sh:113]: I've seen both iterations in scripts, which one do you want me to keep?

**Review by MickLesk — DISMISSED** [review]

File: install/glance-install.sh (hunk 2 above)
[the remark this answers is not in the document]
Comment (kristocopani) [reply install/glance-install.sh]: Done

File: install/glance-install.sh (hunk 5 above)
[the remark this answers is not in the document]
Comment (kristocopani) [reply install/glance-install.sh]: Done

**Comment by kristocopani** [comment]
Made the config yml smaller, added a # on top with the link of documentation!

**Pull request closed without merging**